objectbox/lib/src/relations/to_many.dart

```diff
@@ -105,10 +119,7 @@ class ToMany<EntityT> extends Object with ListMixin<EntityT> {
 
   @override
   void addAll(Iterable<EntityT> iterable) {
-    iterable.forEach((element) {
-      ArgumentError.checkNotNull(element, 'iterable element');
```

Comment:
Element null check removed on purpose? Or would the `_count` map throw because it does not support null keys?

If not, element null check is also missing from new constructor.

Reply:
yes, element cannot be nullable (type doesn't allow that)